### PR TITLE
Task 10 firsrt review

### PR DESCRIPTION
**Pull request by Maksym-Lizohub**

### CODE REVIEW

File: app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
```
Comment (y-mantuliak) [inline app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml:2]: Please add template annotation

File: app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            <ol class="items" data-role="content">
+                <?php
+        $subcats = $viewModel->getSubcategories();
+        foreach ($subcats as $subcat) {
+            if ($subcat->getIsActive()) {
+                $subcat_url = $subcat->getUrl();
+        ?>
+                <li class="item">
+                    <a class="sub-category__link" href="<?php echo $subcat_url; ?>">
+                        <p><?php echo $subcat->getName(); ?></p>
```
Comment (y-mantuliak) [inline app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml]: use $escaper for `$subcat->getName()` and `$subcat_url`

**Review by y-mantuliak — CHANGES_REQUESTED** [review]

File: app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml (hunk 1 above)
Comment (Maksym-Lizohub) [reply app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml:2]: add annotation

Comment (Maksym-Lizohub) [reply app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml:2]: done

File: app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml (hunk 2 above)
Comment (Maksym-Lizohub) [reply app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml]: done

**Review by y-mantuliak — APPROVED** [review]

**Pull request merged**